src/current/_includes/releases/v23.1/v23.1.14.md

```diff
@@ -0,0 +1,100 @@
+- Replica processing in store queues has been enhanced to smoothly handle replicas that have undergone ID changes or destruction. These replicas can now be effectively re-added and processed without encountering previous purgatory-related issues. [#115036][#115036]
+- Fixed an issue that could lead to stuck queries or inaccurate results when using lookup [joins](https://www.cockroachlabs.com/docs/v23.1/joins) involving equality columns and multiple ranges. The bug was introduced in 22.2 and is now fixed. [#115581][#115581]
+- [`CREATE INDEX`](https://www.cockroachlabs.com/docs/v23.1/create-index) with expressions no longer fail on materialized [views](https://www.cockroachlabs.com/docs/v23.1/views) when the declarative schema changer is used. [#115596][#115596]
+- Fixed a bug that could result in an incorrect "too few columns" error for queries that use `ANY <array>` syntax with a subquery. [#115591][#115591]
+- Fixed a bug that could cause "too few/many columns" errors for queries that used `IN` or `NOT IN` with a non-trivial right operand, such as a subquery (rather than a constant tuple). [#115591][#115591]
+- Fixed a bug that could cause errors or inaccurate results when performing lookup and index [joins](https://www.cockroachlabs.com/docs/v23.1/joins) on tables with three or more column families. The bug was introduced in 22.2 and is now fixed. [#115899][#115899]
+- Fixed a bug that could cause internal errors or panics while attempting to forecast statistics on a numeric column. [#115899][#115899]
+- [`BEGIN`](https://www.cockroachlabs.com/docs/v23.1/begin-transaction) statement logs now have correct query `Age` fields. Previously, these fields were being recorded incorrectly, which would lead to some statements also appearing erroneously in the slow query logs. [#115257][#115257]
+- SQL stats in the UI now correctly reflect total runtime, resolving a calculation error that previously led to values exceeding 100%. [#117497][#117497]
+- Custom key column now functions with [change data capture queries](https://www.cockroachlabs.com/docs/v23.1/cdc-queries) correctly. For example, `CREATE CHANGEFEED WITH key_column=..., unordered AS SELECT * FROM table` now works correctly instead of retrying forever. Note that some functionalities with CDC custom keys are not fully supported, see 115267 for more details. [#117555][#117555]
+
+<h3 id="v23-1-14-performance-improvements">Performance improvements</h3>
+
+- Reduced query planning time significantly for some queries which join multiple tables. [#114512][#114512][#116107][#116107]
+
+<div class="release-note-contributors" markdown="1">
+
+<h3 id="v23-1-14-contributors">Contributors</h3>
+
+This release includes 128 merged PRs by 42 authors.
+We would like to thank the following contributors from the CockroachDB community:
+
+- Matthew Sanabria (first-time contributor)
+- rharding6373
```

Comment:
```suggestion
```
Both are CRL employees. (I verify by looking for the last name in Slack, typically)